### ISSUE TITLE
Mongodb sink connector is not inserting in mongodb

### ISSUE THREAD

**Issue by nathgopi45** [RADAR-CNS/RADAR-MongoDbConnector#7]
I have added my sink properties as follows, 
Sink.properties 
name=kafka-connector-mongodb-sink
connector.class=org.radarcns.mongodb.MongoDbSinkConnector
tasks.max=1
buffer.capacity=100
topics=test.test.test
mongo.host=127.0.0.1
mongo.database=test
mongo.port=27017

but when i try to load the sink connectors as follows,
./bin/connect-standalone ./etc/schema-registry/connect-avro-standalone.properties ./etc/kafka/sink.properties

It is always giving me as 
[2017-01-03 09:05:17,993] INFO 0 have been processed initiating MongoDbSinkTask (org.radarcns.mongodb.MongoDbSinkTask:57)
[2017-01-03 09:05:18,514] INFO 0 have been written in MongoDB 0 records need to be processed. (org.radarcns.mongodb.MongoDbWriter:60)

I can see in MongoDbWriter it is executing the following code under run() method,
 while (!stopping.get()) {
//code
}

but my buffer is always empty.. so it is giving the output as above..
I cant see in my logs that nowhere it is calling put() method  which is to fill the buffer from MongoDbSinkTask class. And when starting , it shows zero record have to be processed from initial assessment.

Logs 
[2017-01-03 08:50:17,846] INFO Created connector kafka-connector-mongodb-sink (org.apache.kafka.connect.cli.ConnectStandalone:93)
[2017-01-03 08:50:17,959] INFO 0 have been processed initiating MongoDbSinkTask (org.radarcns.mongodb.MongoDbSinkTask:57)
[2017-01-03 08:50:18,261] INFO Cluster created with settings {hosts=[127.0.0.1:27017], mode=SINGLE, requiredClusterType=UNKNOWN, serverSelectionTimeout='30000 ms', maxWaitQueueSize=500} (org.mongodb.driver.cluster:71)

Kindly let me know the issue in this..or in case if further details needed

**Comment by fnobilia** [comment]
Hi @nathgopi45 thanks for your feedback. We have already planned more tests on this component for this and the next weeks. We'll keep you posted.

**Comment by nivemaham** [comment]
HI @nathgopi45, are you trying to consume multiple topics? In that case they have to be comma separated. ( test1, test2, test3)
Are you using a custom RecordConverterFactory and RecordConverter or using the default one? You have to make sure that there is a suitable RecordConverter available according to your schema.

**Comment by fnobilia** [comment]
Hi @nathgopi45 the bug has been fixed in the dev branch.
Now you will see a log like this
```
INFO 11 have been processed
INFO 11 have been written in MongoDB 0 records need to be processed
INFO [FLUSH-Writer] Time-laps: 147899nsec
INFO [FLUSH] Time-laps: 546823nsec
INFO WorkerSinkTask{id=radar-connector-mongodb-sink-0} Committing offsets
```

I will close the issue after a long-run test.

**Comment by nivemaham** [comment]
@fnobilia Can we close this issue?

**Comment by fnobilia** [comment]
@nivemaham yes please. While closing, could you add a reference to the first release?

**Comment by nivemaham** [comment]
Fixed in release [0.1](https://github.com/RADAR-CNS/MongoDb-Sink-Connector/releases/tag/0.1)
